Submission1

```diff
@@ -0,0 +1,124 @@
+/* ----------------------------------------------------------------------------
+ * Copyright &copy; 2015 Jaemin Jin <jaemin_95@csu.fullerton.edu>
+ * Released under the [MIT License] (http://opensource.org/licenses/MIT)
+ * ------------------------------------------------------------------------- */
+ 
+/**
+ * Write a program that a number converts into a Roman Numeral.
+ */
+ 
+#include <iostream>
+using std::cin;
+using std::cout;
+using std::endl;
+
+#include <string>
+using std::string;
+
+void RomanNumerals(int number1);
```

Comment:
i think the function was supposed to return a `string` rather than output the number directly.  but that's okay.